### ISSUE TITLE
[1.15] experimental_ai only available in debug mode

### ISSUE THREAD

**Issue by sevu** [wesnoth/wesnoth#4997]
An unintended side effect of hiding it, all sides with `[side][ai]ai_algorithm=experimental_ai` have effectively no AI unless started with `-d`.

I also think it doesn't need to be hidden. It has still the different recruiting where it differs from the standard AI, which is a big feature.

This bug was found by @knyghtmare playing a campaign on 1.15, the one I used to test the experimental_ai last year.

**Comment by mattsc** [comment]
I'm not sure whether I agree that this is a bug and a blocker.  If you set the AI algorithm to a "non-existing" AI, that's no surprise that it won't work.

Having said that, I do agree that having the ExpAI hidden does not make much sense given the current differences.  Originally, the new default AI was supposed to be much closer to the ExpAI.  I had planned to port the new recruiting algorithm also, and castle switching was removed from default again because it caused more problems than it helped.

Bottom line is: I'll wait a couple days just in case there are objections (I don't expect any) and then I'll make the ExpAI available again without debug mode.

**Comment by Pentarctagon** [comment]
I've removed the "Blocker: Point" label since this doesn't cause widespread and gamebreaking issues.

**Comment by mattsc** [comment]
Closing via c725376.
